src/Rules/Deprecations/FetchingDeprecatedConstRule.php

```diff
@@ -54,6 +54,24 @@ public function processNode(Node $node, Scope $scope): array
 		}
 
 		$constantReflection = $this->reflectionProvider->getConstant($node->name, $scope);
+		$deprecatedMessage = $constantReflection->getDeprecatedDescription();
+
+		// handle `@deprecated 7.3` as a min php version constraint.
+		// this notation is used in jetbrains/phpstorm-stubs
+		sscanf($deprecatedMessage ?? '', '%d.%d', $phpMajor, $phpMinor);
```

Comment:
we interpret the annotation from phpstorm-stubs

https://github.com/JetBrains/phpstorm-stubs/blob/9608c953230b08f07b703ecfe459cc58d5421437/filter/filter.php#L478

Reply:
This logic isn't needed here at all. This should be handled by PhpStormStubsSourceStubber.